src/css/main.css

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
@shreedharshetty this is the build file. not supposed to be here add it in .gitignore